src/system/AdminModule/Resources/views/AdminInterface/categories.panel.html.twig

```diff
@@ -0,0 +1,59 @@
+{{ pageAddAsset('javascript', zasset('@ZikulaAdminModule:js/jQuery.mmenu-5.5.1/dist/core/js/jquery.mmenu.min.all.js')) }}
+{{ pageAddAsset('stylesheet', zasset('@ZikulaAdminModule:js/jQuery.mmenu-5.5.1/dist/core/css/jquery.mmenu.all.css')) }}
+{{ pageAddAsset('stylesheet', zasset('@ZikulaAdminModule:css/mmenu-hiddenpanel-customwidth.css')) }}
+
+{% macro draw(links, caller, module) %}
+    {% for link in links %}
+        {% if link.links is defined %}
+            <li class="{% if caller._zkModule == module.modname and caller.path == link.url %}Selected{% endif %}">
+                <a href="{{ link.url }}" title="{{ link.text }}"><i class="{{ link.icon }}"></i> {{ link.text }}</a>
+                <ul class="text-left">{{ _self.draw(link.links, caller, module) }}</ul>
+            </li>
+        {% else %}
+            <li class="{% if caller._zkModule == module.modname and caller.path == link.url %}Selected{% endif %}">
+                <a href="{{ link.url }}" title="{{ link.text }}">{% if link.icon is defined %}<i class="{{ link.icon }}"></i> {% endif %}{{ link.text }}</a>
+            </li>
+        {% endif %}
+    {% endfor %}
+{% endmacro %}
+
+{% import _self as menu %}
+
+<a href="#zikula-admin-hiddenpanel-menu"><i class="fa fa-bars"></i></a>
+<nav id="zikula-admin-hiddenpanel-menu">
+    <div class="text-left">
+        <h1><img src="{{ zasset('@ZikulaAdminModule:images/admin.png') }}" alt="{{ __('Administration') }}" /> {{ __('Administration') }}</h1>
+        <ul>
+            {% for category in adminMenu %}
+                <li class="{% if (caller.category.cid) == (category.cid) %} Selected{% endif %}">
+                    <a href="{{ category.url }}">{{ category.title }}</a>
+                    {% if category.modules|length > 0 %}
+                        <ul class="text-left">
+                            {% for module in category.modules %}
+                                <li class="{% if caller.path == module.menutexturl %} Selected{% endif %}">
```

Comment:
class value should be lower-case; also the class attribute can be moved into the condition